### PR TITLE
[Wiley.com] Remove wildcard and update targets

### PR DESCRIPTION
**Pull request by Bisaloo**

### CODE REVIEW

**Comment by Bisaloo** [comment]
This needs to be updated since `wiley.com` updated their site and increased HTTPS coverage.

File: src/chrome/content/rules/Wiley.xml [hunk 1]
```diff
@@ -1,48 +1,65 @@
...
 
-		- bcs		(no https)
-		- eu *
-		- eu.he *
-		- he-cda *
-		- www *
+	Invalid certificate:
+		newsroom.wiley.com
+		*.onlinelibrary.wiley.com
+		cdn.wol2.wiley.com
```
Comment (J0WI) [inline src/chrome/content/rules/Wiley.xml]:
```
admin.wol2.wiley.com
corporatesolutions.wiley.com
news.wiley.com
wileyplus.wiley.com

File: src/chrome/content/rules/Wiley.xml [hunk 2]
```diff
@@ -1,48 +1,65 @@
...
+	<target host="higheredbcs.wiley.com" />
+	<target host="hub.wiley.com" />
+	<target host="olabout.wiley.com" />
+	<target host="onlinelibrary.wiley.com" />
+		<!-- Exclude implicit test URL -->
+		<exclusion pattern="^http://onlinelibrary\.wiley\.com/$" />
+	<target host="ordering.onlinelibrary.wiley.com" />
+		<test url="http://ordering.onlinelibrary.wiley.com/subs.asp?ref=1461-0248&amp;doi=10.1111/(ISSN)1461-0248" />
+	<target host="onlinelibrarystatic.wiley.com" />
+		<test url="http://onlinelibrarystatic.wiley.com/central/prf/UKsprf.pdf" />
```
Comment (J0WI) [inline src/chrome/content/rules/Wiley.xml]:
```
customselect.wiley.com
wolbookstore.wiley.com

File: src/chrome/content/rules/Wiley.xml [hunk 3]
```diff
@@ -1,48 +1,65 @@
...
+		cdn.wol2.wiley.com
 
-	* Times out
+	No working URL known:
+		media.wiley.com
+		secure.onlinelibrary.wiley.com
+		sp.onlinelibrary.wiley.com
 
+	Time out:
+		journalshelp.wiley.com
```
Comment (J0WI) [inline src/chrome/content/rules/Wiley.xml]: `pesgb.wiley.com`

File: src/chrome/content/rules/Wiley.xml [hunk 4]
```diff
@@ -1,48 +1,49 @@
...
+	<target host="he-cda.wiley.com" />
+	<target host="higheredbcs.wiley.com" />
+	<target host="hub.wiley.com" />
+	<target host="media.wiley.com" />
+		<test url="http://media.wiley.com/product_ancillary/4X/08138207/DOWNLOAD/additional%20readings.pdf" />
+	<target host="newsroom.wiley.com" />
+	<target host="onlinelibrary.wiley.com" />
+	<target host="sp.onlinelibrary.wiley.com" />
+		<test url="http://sp.onlinelibrary.wiley.com/Shibboleth.sso/Session" />
+	<target host="onlinelibrarystatic.wiley.com" />
```
Comment (J0WI) [inline src/chrome/content/rules/Wiley.xml]: Some fetch tests failed

**Comment by zoracon** [comment]
Closing due to no revisions made. Can reopen if interest for revision is indicated

**Pull request closed without merging**